src/Product/SearchProvider.php

```diff
@@ -186,7 +186,7 @@ public function runQuery(
 
         $this->labelRangeFilters($facets);
         $this->addEncodedFacetsToFilters($facets);
-        $this->hideUselessFacets($facets, (int) $result->getTotalProductsCount());
+        $this->hideUselessFacets($facets, (int)$result->getTotalProductsCount());
```

Comment:
```suggestion
        if(Configuration::get('PS_LAYERED_HIDE_OTHERS_FILTERS')) {
            $this->hideUselessFacets($facets, (int)$result->getTotalProductsCount());
        }
```

@PierreRambaud I have a problem that I hadn't identified before. It doesn't work with sliders (price, weight). So I suggest this modification which works all the time and to put the initial code back on the hideUselessFacets() function. Ok?

Reply:
I think the best way is to complete the issue I linked in your PR description.
It's about facets filters that don't appear because there is only 1 element.

Reply:
My suggestion also no longer works in the context of a filter that doesn't need to exist, e.g. a category where there would only be one product, this is similar to what I said about your answer about skipping this method. The idea is to keep the filter displayed, only if in the unfiltered list there is more than one product. It is necessary to make an additional condition on the slider. I will make a new commit for this.